alignak/objects/contact.py

```diff
@@ -67,28 +68,28 @@ class Contact(Item):
     """Host class implements monitoring concepts for contact.
     For example it defines host_notification_period, service_notification_period etc.
     """
+    name_property = "contact_name"
     my_type = 'contact'
 
     properties = Item.properties.copy()
     properties.update({
         'contact_name':
             StringProp(fill_brok=['full_status']),
-        'alias':
```

Comment:
Because of inheritance ... alias is a property of an Item which Contacts inherits from